src/libmongoc/src/mongoc/mongoc-stream-tls-secure-channel.c

```diff
@@ -107,7 +107,7 @@ _mongoc_stream_tls_secure_channel_destroy (mongoc_stream_t *stream)
     * Shutting Down an Schannel Connection
     */
 
-   TRACE ("shutting down SSL/TLS connection", NULL);
+   TRACE ("%s", "shutting down SSL/TLS connection");
```

Comment:
This is Windows specific code, so gcc did not warn about this. But I did a mass regex replace. And might as well make it consistent.

Reply:
Ah, I totally missed that `msg` was merely concatenated with the format string in the [`TRACE`](https://github.com/mongodb/mongo-c-driver/blob/master/src/libmongoc/src/mongoc/mongoc-trace-private.h#L39) definition. I thought it was itself an argument.

This seems very sensible and an easy way to fix things.